libs/ledger/src/storage_unit/transaction_store_sync_protocol.cpp

```diff
@@ -132,6 +141,8 @@ uint64_t TransactionStoreSyncProtocol::ObjectCount()
 TransactionStoreSyncProtocol::TxArray TransactionStoreSyncProtocol::PullSubtree(
     byte_array::ConstByteArray const &rid, uint64_t bit_count)
 {
+  telemetry::FunctionTimer telemetry_timer{*pull_subtree_histogram_};
```

Comment:
what does this function do? I would like to understand this telemetry histogram stuff

Reply:
awesome